src/components/Switch/index.tsx

```diff
@@ -273,7 +274,7 @@ const Switch = ({
         ) : null}
         <StyledSwitch
           size={size}
-          aria-checked={checked}
+          data-checked={checked}
```

Comment:
Can't we use `checked={}` directly ?

Reply:
`StyledSwitch` is not an input but a div